netmap/netmap_contract.go

```diff
@@ -216,80 +149,6 @@ func AddPeerIR(nodeInfo []byte) {
 	})
 }
 
-// AddPeer accepts information about the network map candidate in the FrostFS
-// binary protocol format, identifies the caller and behaves depending on different
-// conditions listed below.
```

Comment:
It produces a notification in case `1a`, I don't think we can remove it safely. Have you checked whether it is called in `node`?

Reply:
Yes, but we storage node creates a notary transaction with this method which produces a notary notification (implicit here). I think we should leave this method for now and revisit how we handle node addition/state update later.
cc @alexvanin

Reply:
Changes for `AddPeer` were reverted.